mainapp/templates/account/signup.html

```diff
@@ -21,7 +23,14 @@ <h1>{% trans "Sign Up" %}</h1>
         {% if redirect_field_value %}
         <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
         {% endif %}
-        <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        <div class="row">
+        <div class="col-lg">
+          <button class="btn btn-primary" type="submit">{% trans "Sign Up" %} &raquo;</button>
+        </div>
+        <div class=""col-lg">
+          <button type="button" class="btn btn-primary" onclick="history.back()">Back</button></a>
+        </div>
+        </div>
```

Comment:
What's the reason for this update? I would think that the restaurant API wouldn't have an affect on the user login template.